.github/ISSUE_TEMPLATE/error_template.yml

```diff
@@ -0,0 +1,47 @@
+name: Report an issue
+description: Report an issue you are experiencing.
+labels: General
```

Comment:
@beckermr I just realised I used the same label here as for the general issue – will fix that. What kind of labels are available in this repo to use?

Reply:
Here is [the list of labels]( https://github.com/conda-forge/conda-forge.github.io/labels ). That said, I doubt we want to use all of those, but some reasonable subset might make sense

Reply:
Ahh thanks! Didn't know you could see them not being a member.

PR is in https://github.com/conda-forge/conda-forge.github.io/pull/1594